pyprusalink/__init__.py

```diff
@@ -20,57 +38,19 @@ class Conflict(PrusaLinkError):
     """Error to indicate the command hit a conflict."""
 
 
-class VersionInfo(TypedDict):
-    """Version data."""
-
-    api: str
-    server: str
-    text: str
-    hostname: str
-
-
-class PrinterInfo(TypedDict):
-    """Printer data."""
-
-    telemetry: dict
-    temperature: dict
-    state: dict
-
-
-class JobInfo(TypedDict):
-    """Job data."""
-
-    state: str
-    job: dict | None
-
-
-class FileInfo(TypedDict):
-    """File data."""
-
-    name: str
-    origin: str
-    size: int
-    refs: dict
-
-
-class FilesInfo(TypedDict):
-    """Files data."""
-
-    files: list[FileInfo]
-
-
 class PrusaLink:
```

Comment:
Do we need to split this into multiple classes, one for each version of the API? We can put the request function into a single base class.